tests/metagen/typegraphs/identities/py/fdk.py

```
@@ -0,0 +1,1920 @@
+        parsed = json.loads(res.body)
+        if parsed.get("errors"):
+            raise Exception("graphql errors in response", parsed)
+        return parsed["data"]
+
+    @abstractmethod
+    def fetch(
+        self,
+        doc: str,
+        variables: typing.Dict[str, typing.Any],
+        opts: GraphQLTransportOptions | None,
+        files: typing.Dict[str, File] = {},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

(B006)